Tests/KeyboardToolbarTests.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  KeyboardToolbarTests.swift
+//  KeyboardSupport-iOSTests
+//
+//  Created by Earl Gaspard on 7/31/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import XCTest
+@testable import KeyboardSupport
+
+class KeyboardToolbarTests: XCTestCase {
+    
+    func test_KeyboardToolbar_InitWithFrame() {
+        let keyboardToolbar = KeyboardToolbar()
+        
+        XCTAssertNotNil(keyboardToolbar)
+        XCTAssertEqual(keyboardToolbar.frame.height, 50)
```

Comment:
Test fails because the implicit frame inside the KeyboardToolbar constructor is not present in this version.